documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -2759,6 +2759,27 @@ the behavior seen by the client it is the same that we would have with an optimi
 
 For more information about HotRod transaction see the http://infinispan.org/docs/dev/user_guide/user_guide.html#hot_rod_transaction[Infinispan user guide].
 
+==== Infinispan Remote Stored Procedures
+
+Since version __5.4.0.Beta2__, stored procedures are supported by the remote dialect too.
```

Comment:
I will change this to `The Infinispan remote dialect supports stored procedures since version 5.4.Beta2`.
@fax4ever Is that alright?

Reply:
Actually, I think I will remove it. I don't think we usually add this kind of information in the documentation

Reply:
it makes sense